docker-compose.yml

```diff
@@ -42,6 +42,7 @@ services:
       - ./database/snapshots:/snapshots
     ports:
       - 5431:5432
+
```

Comment:
Adding a line feed here between these two services.

I was in here to try to get the replication working with the arguments to the postgres service, but no luck. I worked around it by commenting out the `REFRESH MATERIALIZED TABLE` command in my saved snapshot, importing it, and then running that refresh manually.

Reply:
bummer! i will get that issue prioritized to put a permanent fix in place.